### ISSUE TITLE
2.0.0 BlurTransformation crashes on Android 4.2

### ISSUE THREAD

**Issue by wolfemm** [wasabeef/glide-transformations#35]
The blur transformation in 2.0.0 crashes the app in my emulator running Android 4.2:

```
04-12 10:57:42.616 1987-2003/jp.wasabeef.example.glide E/RenderScript: rsi_ScriptIntrinsicCreate 5
04-12 10:57:42.616 1987-2003/jp.wasabeef.example.glide A/libc: Fatal signal 11 (SIGSEGV) at 0x00000000 (code=128), thread 2003 (fifo-pool-threa)
```

This happens both in the example app and my own app. Only using RSBlur on Android 4.3+ (https://github.com/wolfemm/glide-transformations/commit/a6cbe3c0d6412679d89d7d7a4a03eef99475773e) seems to be a fix, but I'm not knowledgeable enough to know whether that's the proper solution or not.

**Comment by wasabeef** [comment]
@wolfemm 
Thanks

**Comment by KennyGoers** [comment]
This same issue us occurring for me in 2.1.1, is there something about the fix I missed?

**Comment by wasabeef** [comment]
Are you used the picasso-transformations?
Then, I just released v2.1.2 of picasso-transformations. please, you can try it.

**Comment by KennyGoers** [comment]
Oops, I googled and misread the GitHub project. Thanks!
